packages/workspace/src/utils/rules/format-files.ts

```diff
@@ -98,3 +106,34 @@ function updateWorkspaceJsonToMatchFormatVersion(
     console.error(e);
   }
 }
+
+function objectSort(originalObject: object) {
+  return Object.keys(originalObject)
+    .sort()
+    .reduce((obj, key) => {
+      obj[key] = originalObject[key];
+      return obj;
+    }, {});
+}
```

Comment:
I had to duplicate this function here because the pre-commit check was failing when trying to generate the documentation. The documentation file is importing something from `../../packages/workspace` and it didn't like that this file was importing from `@nrwl/tao/src/utils/object-sort` 🤷🏼‍♂️

Reply:
`ast-utils.ts` already has the same functionality, though.
https://github.com/nrwl/nx/blob/dd45382252f365ba7a8741bf10bd62f6effd9ac2/packages/workspace/src/utils/ast-utils.ts#L67-L76

Reply:
oh that's good then. I don't have to duplicate that function there. 😄 

 Unfortunately, we still need to have the other objectSort that I put in `tao` so that it's compatible without @angular/devkit